docs/widgets.md

```diff
@@ -226,6 +226,233 @@ public class BootstrapTokenSigner {
     return signedJwt.serialize();
   }
 }
+```
+
+  </TabItem>
+
+  <TabItem label="Golang" value="go" default>
+
+Here is a golang snippet to generate a **bootstrap token** for the [crypto on-ramp](./flows/crypto-onramp.mdx) flow using the [`github.com/lestrrat-go/jwx/v2`](https://github.com/lestrrat-go/jwx) package. Please note that the code below is for illustrative purposes only, and the integration should be adapted to your application.
+
+```go
+package main
+
+import (
+	"time"
```

Comment:
We're using tabs in this example, but spaces in all the other examples. Can we change to spaces here please?